src/plugins/console/public/application/containers/editor/legacy/console_editor/keyboard_shortcuts.ts

```diff
@@ -28,39 +28,44 @@ export function registerCommands({
 
   coreEditor.registerKeyboardShortcut({
     keys: { win: 'Ctrl-Enter', mac: 'Command-Enter' },
-    name: 'send to Elasticsearch',
+    name: '__console_send_to_Elasticsearch',
```

Comment:
@sabarasaba I've changed the implementation a bit. Instead of prefixing the every command with `__console`, what if we define the commands that need to be disabled in `unregiterKeyboardShortcuts`. See [here](https://github.com/elastic/kibana/pull/128887/files#diff-ca1f078bd63c665b23b0cadf62c1bc384efb2a8039169aaa5ca6a483a0e28bffR83). WDYT?

Reply:
love it, i think it s a lot clearer now!